qcodes/utils/helpers.py

```diff
@@ -397,8 +396,8 @@ def __getattr__(self, key):
             "'{}' object and its delegates have no attribute '{}'".format(
                 self.__class__.__name__, key))
 
-    def __dir__(self):
-        names = super().__dir__()
+    def __dir__(self) -> List[str]:
```

Comment:
`sorted` returns an iterable, right?
```suggestion
    def __dir__(self) -> Iterable[str]:
```

Reply:
No it always returns a list https://github.com/python/typeshed/blob/master/stdlib/3/builtins.pyi#L1205

Reply:
woooooow! i sohuld've known :)

Reply:
I think this is because you cant sort anything without looking at all the elements so its hard to make it a real Iterable

Reply:
indeed, this makes total sense.